### DISCUSSION TITLE
Java Not In Environment Variables

### DISCUSSION THREAD

**Discussion by Antonimo-Dev** [freeplane/freeplane#2158] [Help]
Hi,

I have a clean installation of Windows 11 Pro 24H2 and I have installed Freeplane-Setup-with-Java-1.12.6. JAVA does not appear in the Environment Variables after this installation.  Should it?

Also, I notice I am unable to drag and drop a *.mm file onto an open Freeplane window to open it. Is this due to the Java Environment variable, or is there another cause?

**Comment by dpolivaev** [comment]
Java should not appear in the environment because it is contained in Freeplane installation and it is only used by Freeplane.
Drag and drop works on my machine fine, I checked it using the latest version 1.12.8

**Comment by Antonimo-Dev** [comment]
Hi Dimitry,

Thanks for the clarification re Java and for confirming that drag and drop works in 1.12.8.

I found the issue was at my end. I was launching Freeplane using a script run as administrator and when I changed the script, I was able to drag and drop again.
